pwnlib/context/__init__.py

```diff
@@ -287,6 +290,9 @@ class ContextType(object):
 
     Examples:
 
+    .. doctest::
```

Comment:
I believe that this will break the formatting under "Examples".  Have you looked at the output of `make -C docs html` to see if the output changes or additional formatting warnings are generated?

Reply:
I haven't yet. According to sphinx doc,

```
Some text

>>> print("Inline example")
Inline example
```

Is equivalent to

```
Some text

.. doctest::

   >>> print("some inline example")
   some inline example
```